.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file from the PR branch, I've also added it to `.gitignore` in the `main` branch, so it won't be included in the repository in the future.

Reply:
Ok. I removed in commit [b159d6b](https://github.com/CoreOffice/OLEKit/pull/9/commits/b159d6b6552e778b4f05c57f64a3879273bf326e)